react/components/atoms/lightbox/lightbox.js

```diff
@@ -39,8 +57,29 @@ export class Lightbox extends PureComponent {
 
     constructor(props) {
         super(props);
+
+        this.screenWidth = Dimensions.get("window").width;
+        this.screenHeight = Dimensions.get("window").height;
+
+        this.fullZoomedInValue = this.props.maxZoomScale;
+        this.fullZoomedOutValue = 1;
+
+        this.translatedX = 0;
+        this.translatedXTreshold = 0;
+        this.translatedY = 0;
+        this.translatedYTreshold = 0;
+
+        this.baseScale = new Animated.Value(1);
+        this.scaleRate = new Animated.Value(1);
+        this.lastScale = 1;
+
         this.state = {
-            visible: false
+            zoomed: false,
+            zooming: false,
+            visible: this.props.visible,
+            scale: Animated.multiply(this.baseScale, this.scaleRate),
+            translateX: new Animated.Value(0),
+            translateY: new Animated.Value(0)

```

Comment:
You can remove these from state since they are animations, so that they can stay together with the other 2 variables mentioned above